README.md

```diff
@@ -221,6 +221,9 @@ Use the searchandiser method to set up your configurations in a script tag at th
         //
         // hide 'first' and 'last' links
         // showTerminals: false
+        //
```

Comment:
this isn't exactly accurate, it will use the page numbers *instead* of using the page labels which default to `First` and `Last`